data/json/recipes/recipe_weapon.json

```diff
@@ -1368,6 +1368,31 @@
   ]
 },{
   "type" : "recipe",
+  "result": "pipe_combination_gun",
```

Comment:
Don't we have enough materials to make this recipe a little more interesting?
The pipes and wood cover the barrels and stock, the scrap covers a trigger assembly, but you need a good spring for the firing pin, which speaking of should probably be a few nails just for ambiance, then you need the breech, that needs to be a block of metal, not just some random scrap, I guess scrap covers misc screws as well, but it'd be better if we had inventories of screws that get used up by recipes like these.  Additionally you need something to bind the whole mess together, rope, duct tape, something.